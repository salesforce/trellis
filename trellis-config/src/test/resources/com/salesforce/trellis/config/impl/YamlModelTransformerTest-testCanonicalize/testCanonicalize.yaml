groups:
- name: PE_IMPLS
  includes:
  - sfdc.shared:baz
  - sfdc.core:platform-encryption
  - sfdc.shared:bar
  - sfdc.ui:*
  - sfdc.shared:foo

- name: PE_APIS
  includes:
  - sfdc.core:platform-encryption-api

rules:
- action: DENY
  from: [PE_APIS]
  to: [PE_IMPLS]
  reason: hello!
