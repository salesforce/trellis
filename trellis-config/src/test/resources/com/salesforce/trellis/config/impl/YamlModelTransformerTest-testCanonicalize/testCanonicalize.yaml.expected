groups:

- name: PE_APIS
  includes:
  - sfdc.core:platform-encryption-api

- name: PE_IMPLS
  includes:
  - sfdc.core:platform-encryption
  - sfdc.shared:bar
  - sfdc.shared:baz
  - sfdc.shared:foo
  - sfdc.ui:*

rules:

- action: DENY
  from:
  - PE_APIS
  to:
  - PE_IMPLS
  reason: hello!
